# Translations template for glance.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the glance project.
#
# Translators:
# Adriana Chisco Landazábal <achisco94@gmail.com>, 2015
msgid ""
msgstr ""
"Project-Id-Version: Glance\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2015-06-24 06:10+0000\n"
"PO-Revision-Date: 2015-06-23 13:20+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: Spanish (http://www.transifex.com/projects/p/glance/language/"
"es/)\n"
"Language: es\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#, python-format
msgid ""
"%(image_id)s: field %(key)s differs (source is %(master_value)s, destination "
"is %(slave_value)s)"
msgstr ""
"%(image_id)s: campo %(key)s difiere (fuente es %(master_value)s, destino es "
"%(slave_value)s)"

#, python-format
msgid "%s directory does not exist."
msgstr "No existe directorio %s."

#, python-format
msgid "Access denied to image %(id)s but returning 'not found'"
msgstr "Acceso denegado a la imagen %(id)s aunque se devuelve 'not found'"

#, python-format
msgid "An optional task has failed, the failure was: %s"
msgstr "Ha fallado una tarea opcional, el error fue: %s"

#, python-format
msgid "Artifact with id=%s is not accessible"
msgstr "No se puede acceder al artefacto con id=%s"

#, python-format
msgid "Artifact with id=%s not found"
msgstr "No se encontró artefacto con id=%s"

msgid "Artifact with the specified type, name and version already exists"
msgstr "Ya existe artefacto con el tipo, nombre  versión especificada"

#, python-format
msgid ""
"Artifact with the specified type, name and version already has the direct "
"dependency=%d"
msgstr ""
"El artefacto con el tipo, nombre  versión especificada ya tiene la "
"dependencia directa=%d"

#, python-format
msgid ""
"Artifact with the specified type, name and versions already has the direct "
"dependency=%s"
msgstr ""
"El artefacto con el tipo, nombre  versión especificada ya tiene la "
"dependencia directa=%s "

msgid "Attempted to modify image user did not own."
msgstr ""
"Se ha intentado modificar una imagen que no era propiedad del usuario. "

#, python-format
msgid "Cached image file '%s' doesn't exist, unable to delete"
msgstr ""
"El archivo de imagen almacenado en memoria caché '%s' no existe, no se puede "
"suprimir"

#, python-format
msgid "Can't load artifact %s: load disabled in config"
msgstr ""
"No se puede cargar artefacto %s: la carga no está habilitada en la "
"configuración"

#, python-format
msgid "Can't load artifact %s: not in available_plugins list"
msgstr "No se puede cargar artefacto %s: no en lista available_plugins"

#, python-format
msgid "Could not find image %s"
msgstr "No se ha podido encontrar la imagen %s"

#, python-format
msgid ""
"Could not find schema properties file %s. Continuing without custom "
"properties"
msgstr ""
"No se ha podido encontrar el archivo de propiedades del esquema %s. Se va a "
"continuar sin las propiedades personalizadas"

#, python-format
msgid "Could not find task %s"
msgstr "No se encontró tarea %s"

#, python-format
msgid "Could not find task info %s"
msgstr "No se encontró información de tarea %s"

msgid "Deadlock detected. Retrying..."
msgstr "Punto muerto detectado. Intentando de nuevo..."

#, python-format
msgid "Duplicate entry for values: %s"
msgstr "Conexión para valores duplicada: %s"

#, python-format
msgid ""
"Failed to activate image %s in registry. About to delete image bits from "
"store and update status to 'killed'."
msgstr ""
"No se pudo activar imagen %s en registro. A punto de eliminar bits de imagen "
"del almacen y  actualizar estado a 'killed'."

#, python-format
msgid "Failed to decrypt location value for image %(image_id)s"
msgstr "Error al decodificar valor de ubicación para imagen %(image_id)s"

#, python-format
msgid "Failed to delete blob %s in store from URI"
msgstr "No se eliminó objeto %s en almacén de URI"

#, python-format
msgid "Failed to delete file %(path)s. Got error: %(e)s"
msgstr "No se ha eliminado fichero %(path)s. Tuvo error: %(e)s"

#, python-format
msgid "Failed to delete image %s in store from URI"
msgstr "No se eliminó imagen %s en almacén de URI"

#, python-format
msgid "Failed to find image to delete: %s"
msgstr "No se ha encontrado la imagen para eliminar: %s"

#, python-format
msgid "Failed to find task %(task_id)s. Reason: %(reason)s"
msgstr "No se encontró tarea %(task_id)s. Razón: %(reason)s"

msgid "Failed to successfully cache all images in queue."
msgstr ""
"No se ha podido almacenar en memoria caché satisfactoriamente todas las "
"imágenes en cola."

#, python-format
msgid ""
"Fetch of cache file failed (%(e)s), rolling back by moving "
"'%(incomplete_path)s' to '%(invalid_path)s'"
msgstr ""
"Se ha encontrado un error en la captación del archivo de memoria caché "
"(%(e)s), se va a retrotraer moviendo '%(incomplete_path)s' a "
"'%(invalid_path)s'"

#, python-format
msgid "Forbidden to create task. Reason: %(reason)s"
msgstr "Se olvidó crear tarea. Razón: %(reason)s"

#, python-format
msgid "Forbidden to get task %(task_id)s. Reason: %(reason)s"
msgstr "Se olvidó obtener tarea %(task_id)s. Razón: %(reason)s"

msgid "Id not in sort_keys; is sort_keys unique?"
msgstr "Id no está en sort_keys; ¿es sort_keys exclusivo?"

#, python-format
msgid "Image %s entirely missing from the destination"
msgstr "Falta la totalidad de la imagen %s del destino"

#, python-format
msgid "Image '%s' is not active. Not caching."
msgstr "La imagen '%s' no está activa. No almacenando en memoria caché. "

#, python-format
msgid ""
"Image cache driver '%(driver_module)s' failed to configure. Got error: "
"'%(config_err)s"
msgstr ""
"El controlador de caché de imágenes '%(driver_module)s' no se ha podido "
"configurar. Se ha obtenido el error: '%(config_err)s"

#, python-format
msgid ""
"Image cache driver '%(driver_name)s' failed to load. Got error: "
"'%(import_err)s."
msgstr ""
"El controlador de caché de imágenes '%(driver_name)s' no se ha podido "
"cargar. Se ha obtenido el error: '%(import_err)s."

#, python-format
msgid "Invalid marker. Image %(id)s could not be found."
msgstr "Marcador no válido. No se puede encontrar imagen %(id)s."

#, python-format
msgid "Invalid membership association specified for image %(id)s"
msgstr ""
"Se ha especificado una asociación de pertenencia no válida para la imagen "
"%(id)s"

#, python-format
msgid "Member %(id)s not found"
msgstr "No se ha encontrado el miembro %(id)s"

#, python-format
msgid "Metadata tag not found for id %s"
msgstr "No se ha encontrado etiqueta de metadatos para id%s"

#, python-format
msgid "No metadata found for image '%s'"
msgstr "No se han encontrado metadatos para la imagen '%s' "

#, python-format
msgid "Show level %s is not supported in this operation"
msgstr "EL nivel de vista %s no es soportado por esta operación"

#, python-format
msgid "Task %(task_id)s failed with exception %(error)s"
msgstr "Tarea %(task_id)s falló con exceepción %(error)s"

msgid "The `eventlet` executor has been deprecated. Use `taskflow` instead."
msgstr ""
"Se ha desaprobado el uso del ejecutor `eventlet`. En su lugar use `taskflow`."

#, python-format
msgid ""
"The conversion format is None, please add a value for it in the config file "
"for this task to work: %s"
msgstr ""
"No hay formato de conversión, por favor agregue un valor para éste en el "
"fichero de configuración para esta tarea en el trabajo: %s"

#, python-format
msgid "Unable to delete URI from image %s."
msgstr "No se puede eliminar URI de imagen %s."

msgid "Unable to get deleted image"
msgstr "No se ha podido obtener una imagen suprimida"

#, python-format
msgid "Unable to get deleted task %s"
msgstr "No se puede obtener tarea eliminada %s"

msgid "Unable to get unowned image"
msgstr "No se ha podido obtener una imagen que no es de su propiedad"

msgid "Unknown version. Returning version choices."
msgstr "Versión desconocida. Se van a devolver las opciones de versión."

#, python-format
msgid "Unrecognised child %s"
msgstr "Hijo %s no reconocido"

#, python-format
msgid ""
"User %(user)s attempted to upload an image of size %(size)d that will exceed "
"the quota. %(remaining)d bytes remaining."
msgstr ""
"Usuario %(user)s intentó cargar una imagen de tamaño %(size)d que excede la "
"capacidad. %(remaining)d bytes restantes."

#, python-format
msgid ""
"User %(user)s attempted to upload an image of unknown size that will exceed "
"the quota. %(remaining)d bytes remaining."
msgstr ""
"Usuario %(user)s intentó cargar una imagen de tamaño desconocido que excede "
"la capacidad. %(remaining)d bytes restantes."

#, python-format
msgid "User lacks permission to share image %(id)s"
msgstr "El usuario no tiene permiso para compartir la imagen %(id)s"
